blueprints/serverless/cloud-run-explore/main.tf

```diff
@@ -49,7 +49,10 @@ module "cloud_run" {
     }
   }
   iam = {
-    "roles/run.invoker" = ["allUsers"]
+    "roles/run.invoker" = (local.gclb_create && var.iap.enabled
+      ? ["serviceAccount:${google_project_service_identity.iap_sa[0].email}"]
```

Comment:
did you try this branch? Terraform has the nasty habit of checking both branches regardless of condition, and this will fail if the resource does not exist; it got better lately, but I would wrap this into a `try/null` just to be extra safe

Reply:
I tried because I was not sure of the terraform behavior (I expected it to be the opposite :/ ).  But now I realize I didn't check it correctly. Thanks, fixed.